libs/features/placeholders.js

```diff
@@ -3,8 +3,24 @@ import { customFetch, getConfig, getMetadata } from '../utils/utils.js';
 const fetchedPlaceholders = {};
 window.mph = {};
 
+export const getPlaceholderRoot = (config) => {
+  const placeholderConstentRoot = getMetadata('placeholders-content-root');
+  if (!placeholderConstentRoot) return config.locale?.contentRoot;
+
+  let origin = 'https://www.adobe.com';
```

Comment:
```suggestion
  let origin = window.location.origin;
```

this will ensure placeholders are pulled from 'https://www.stage.adobe.com' when on stage

Reply:
There is placeholder-stage in case stage is on.
Using prod URL as default is more making sense since the file isn't affected by code.

Reply:
If `placeholders-repo` is not defined, but `placeholders-content-root` is, then there will be unexpected behavior that may confuse authors.  Both are required for this to work, so what do you think about defining both values in one metadata field?  Something like:
`placeholders-override: reponame,placeholder-content-root`
`placeholders-override : cc,cc-shared`

Also in the future there may be other consumer projects on non-www subdomains that want to use this, but for now I think we can leave as www.adobe.com.